src/components/Button.tsx

```diff
@@ -82,18 +83,19 @@ export default class Button extends React.PureComponent<{
 				disabled={disabled}
 				onPress={onPress}
 				testID={testID}
-				style={[finalButtonStyles, style]}
```

Comment:
Why change this, in this case in ios app the bottom buttton won't stick to the bottom. For example in the `PinInput`View.

Reply:
Oh! Wow, I thought you've just reduced amount of boxes the there. Should've asked